style.css

```diff
@@ -0,0 +1,324 @@
+}
+
+.header a.myButton {
+  background-color: #04391d;
+  color: white;
+  margin-left: 12px;
+  margin-right: 20px;
+}
+
+.header a.myButton:hover {
+background-color:#032111;
+}
+
+#banner img {
+  height: 300px;
+  width: 100%;
+  object-fit: cover;
+  object-position: 50%;
+}
+
+@media screen and (min-width: 668px) {
+  .header a {
+    float: left;
+    text-align: center;
+  }
+  
+  .header-right {
+    float: right;
+  }
+
+  #banner img {
+    height: 500px;
+    width: 100%;
+  }
+}
+
+
+main{
+ flex-grow: 1;
+}
+
+h1{
+color: #04391d;
+font-size: 38px;
+font-family: Arial, Helvetica, sans-serif;
+text-transform: uppercase;
+}
+
+h2{
+  color: #04391d;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+  text-transform: uppercase;
+  }
+
+h3{
+  color: #04391d;
+  font-size: 22px;
+  font-family: Arial, Helvetica, sans-serif;
+  text-transform: uppercase;
+}
+
+p {
+    font-size: 20px;
+}
+
+.button-document{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+a.button-document {
+  width: 200px;
+  background-color: #989c9a;
+  color: black;
+  padding: 12px;
+  text-decoration: none;
+}
+
+a.button-document:hover {
+  width: 200px;
+  background-color: #575958;
+  color: black;
+  padding: 12px;
+  text-decoration: none;
+}
+
+
+.gallery1 {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 10px;
+}
+
+.gallery1-1 img{
+  width: 450px;
+  height: 300px;
+  object-fit: cover;
+  object-position: 50%
+}
+
+.gallery1-1 {
+width: 250px;
+margin: 0 20px;
+box-shadow: 0 0 20px 2px rgba(0, 0, 0, .1);
+transition: 1s;
+}
+
+.gallery1-1:hover{
+transform: scale(1.3);
+z-index:2;
+}
+
+@media screen and (min-width: 668px) {
+.gallery1 {
+  flex-direction: row;
+  justify-content: center;
+}
+
+.gallery1-1 img{
+  width: 250px;
+  height: 200px;
+  object-fit: cover;
+  object-position: 50%
+}
+
+.gallery1-1 {
+  width: 250px;
+  margin: 0 10px;
+  box-shadow: 0 0 20px 2px rgba(0, 0, 0, .1);
+  transition: 1s;
+  }
+}
+
+#welcome {
+  margin: 40px;
+}
+
+
+#food {
+  margin: 40px
+}
+
+#drinks {
+  margin: 40px;
+}
+
+#team {
+  margin: 40px
+}
+
+
+.box {
+  border-style: solid;
+  border-color: #04391d;
+  margin: 40px;
+  padding: 20px;
+  display: flex;
+  flex-direction: row;
+  row-gap: 20px;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.worker {
+  padding: 5px;
+}
+
+
+.box img {
+max-width: 200px;
+border-radius: 100%;
+-webkit-border-radius: 100%;
+-moz-border-radius: 100%;
+padding: 10px;
+margin: 10px;
+background: white;
+border-color: #04391d;
+border: 2px;
+border-style: solid;
+background-color: #04391d;
+}
+
+.box img:hover{
+  max-width: 200px;
+  border-radius: 100%;
+  -webkit-border-radius: 100%;
+  -moz-border-radius: 100%;
+  padding: 10px;
+  margin: 10px;
+  background: white;
+  border-color: #04391d;
+  border: 2px;
+  border-style: solid;
+  background-color: #ffffff;
+}
```

Comment:
Nice hover effect here too!
Just remember that you only need to specify the things that are different from you original styling